packages/visx-bounds/test/withBoundingRects.test.tsx

```diff
@@ -40,41 +68,48 @@ describe('withBoundingRects()', () => {
     expect(withBoundingRects).toBeDefined();
   });
 
-  test('it should pass rect, parentRect, and getRect props to the wrapped component', () => {
-    const Component = () => <div />;
-    const HOC = withBoundingRects(Component);
-    const wrapper = mount(<HOC />);
-    const RenderedComponent = wrapper.find(Component);
+  test('it should pass rect, parentRect, and getRect props to the wrapped component', async () => {
+    const HOC = withBoundingRects(BoundingRectsComponent);
+    // @ts-ignore
+    const { getByTestId } = render(<HOC />);
+
+    // getBoundingClientRect should be called twice, once for the component, and once for its parent
+    await waitFor(() => expect(Element.prototype.getBoundingClientRect).toHaveBeenCalledTimes(2));
+
+    const RenderedComponent = await getByTestId('BoundingRectsComponent');
+    const RenderedComponentParent = await getByTestId('BoundingRectsComponentParent');
 
-    expect(Element.prototype.getBoundingClientRect).toHaveBeenCalled();
-    expect(RenderedComponent.prop('rect')).toEqual(expectedRectShape);
-    expect(RenderedComponent.prop('parentRect')).toEqual(expectedRectShape);
```

Comment:
@williaster This is where RTL's philosophy of testing differs from Enzyme. [https://testing-library.com/docs/#what-you-should-avoid-with-testing-library](Directly checking internal props & states is a NO-NO in RTL)

Reply:
^I agree the RTL philosophy is better for things like this 💯 this is why enzyme struggles with adapters for each react release 😅